src/main/java/com/SAS/game_event_logger/YellowTicket.java

```diff
@@ -0,0 +1,4 @@
+package com.SAS.game_event_logger;
+
+public class YellowTicket extends Ticket {
```

Comment:
Maybe the type - yellow or red should be as enum?

Reply:
we should concider it later after we have more Use Cases and see what is more easy